src/main/java/com/amazonaws/services/neptune/rdf/io/NeptuneExportSparqlRepository.java

```diff
@@ -41,6 +57,22 @@ public NeptuneExportSparqlRepository(String endpointUrl, AWSCredentialsProvider
         this.awsCredentialsProvider = awsCredentialsProvider;
         this.regionName = regionName;
         this.initAuthenticatingHttpClient();
+
```

Comment:
How does the client builder/http session manager work when you have IAM auth? Will this new `clientBuilder` override the auth client set by `initAuthenticatingHttpClient()`?

Reply:
Good catch. I changed where I add the interceptor to fix any IAM incompatibilities.